frontend/app/routes/$lang/_public/apply/$id/child/review-child-information.tsx

```diff
@@ -76,18 +78,12 @@ export async function loader({ context: { session }, params, request }: LoaderFu
         acessToDentalInsurance: child.dentalInsurance,
         federalBenefit: {
           access: child.dentalBenefits.hasFederalBenefits,
-          benefit: allFederalSocialPrograms
-            .filter((obj) => obj.id === child.dentalBenefits.federalSocialProgram)
-            .map((obj) => getNameByLanguage(locale, obj))
-            .join(', '),
+          benefit: selectedFederalBenefit ? selectedFederalBenefit.name : '',
         },
         provTerrBenefit: {
           access: child.dentalBenefits.hasProvincialTerritorialBenefits,
           province: child.dentalBenefits.province,
-          benefit: allProvincialTerritorialSocialPrograms
-            .filter((obj) => obj.id === child.dentalBenefits.provincialTerritorialSocialProgram)
-            .map((obj) => getNameByLanguage(locale, obj))
-            .join(', '),
+          benefit: selectedProvincialBenefit ? selectedProvincialBenefit.name : '',
```

Comment:
Use `undefined` when `selectedProvincialBenefit` is `undefined` else use `name`. Same logic in other places.
```suggestion
          benefit: selectedProvincialBenefit && selectedProvincialBenefit.name,
```